staging/src/k8s.io/client-go/examples/informer/README.md

```diff
@@ -0,0 +1,34 @@
+# Informer Example
+
+Informers provide a high-level API for creating custom controllers for Kubernetes resources.
+
+This particular example demonstrates:
+
+* How to write an Informer against a core resource type.
+* How to handle add, update and delete events.
+
+The [thirdparty-resources example](../third-party-resources) demonstrates how to use informers against a thirdparty resource.
+
+## Running
+
+To run the example outside the Kubernetes cluster you need to supply the path to a Kubernetes config file.
+
+```sh
+go run main.go -kubeconfig=$HOME/.kube/config -logtostderr
+```
+
+By default `glog` logs to files.
+Use the `-logtostderr` command line argument so that you can see the output on the console.
+
+## Running Inside a Kubernetes Cluster
+
+You can also run the example inside a Kubernetes cluster.
+In this case it will use [in Cluster configuration](../in-cluster/),
+and you don't need to supply `-kubeconfig` or `-master` command line flags.
+
+## Use Cases
+
+* Building controllers that coordinate other resources.
+  Most controllers in [k8s.io/kubernetes/pkg/controller](https://godoc.org/k8s.io/kubernetes/pkg/controller) use informers.
+* Capturing resource events for logging to external systems
+  (e.g. monitor non-"Normal" events and publish metrics to a time series database)
```

Comment:
Can you add a `## Cleanup` section people can run to clean up the artifacts created by running this example? (For instance: https://github.com/kubernetes/client-go/tree/master/examples/create-update-delete-deployment or https://github.com/kubernetes/client-go/tree/master/examples/in-cluster has this section). 

This will help us keep the README files for each client-go example similar.